### ISSUE TITLE
Getting started...

### ISSUE THREAD

**Issue by zaarheed** [wearede/autoblog-addon-full-text-importer#1]
Apologies in advance for my ignorance.

I have activated the plugin from the Autoblog dashboard. The site I would like to import the full post text from has post pages that have the body of content within the 'entry-content' class.

I have added the RSS link as the source, and set the class to entry-content - but now Autoblog is pulling just the title. Without your plugin it would pull the title and the excerpt.

Am I doing something wrong? Any pointers on how to get started?

**Comment by sandrodz** [comment]
Well that should be enough for the plugin. Check if entry-content exists in source and its not added via js.

**Comment by zaarheed** [comment]
Yes, the entry-content class exists in the source - however it is a nested class, does that make a difference?

Have you tested this against importing the full text from other Wordpress sites using the default RSS feed?

Do you have a live demo uploaded anywhere that I can inspect? Otherwise, I'll have a deeper look into your source file to see if I can find any hints as to why it may not be working.

**Comment by sandrodz** [comment]
Nesting doesn't make a difference. Plugin grabs everything inside the  target class and doesn't care about its parents.

Yes of course, it's been tested on several blogs, it doesn't really care if its wp or some other cms. Its important to have full article link and class for container.

In fact I do, but blogs are not english, I don't think you will find them particularly useful.
